extensions/amp-iframe/amp-iframe.md

```diff
@@ -243,7 +241,8 @@ window.addEventListener('message', function (event) {
 
 The intersection message would be sent by the parent to the iframe in the format of IntersectionObserver entry wheneve there is intersectionRatio change across thresholds [0, 0.05, 0.1, ... 0.9, 0.95, 1].
 
-## Iframe & Consent Data
+<!-- TODO: uncomment when Bento amp-consent (v1.0) is ready -->
```

Comment:
Let's create an issue or something to refer to here?

Reply:
Done. Created #36211 & linked in todo.